internal/source/iterator.go

```diff
@@ -53,8 +57,8 @@ type IteratorParams struct {
 	AckPolicy      nats.AckPolicy
 }
 
-// getSubscribeOptions returns a NATS subscribe options based on the IteratorParams's fields.
-func (p IteratorParams) getSubscribeOptions() ([]nats.SubOpt, error) {
+// generateConsumerConfig returns a NATS subscribe options based on the IteratorParams's fields.
+func (p IteratorParams) getSubscriberOpts(ctx context.Context) ([]nats.SubOpt, error) {
```

Comment:
Nitpick: the comment mentions a different function name.

Reply:
Good catch!